src/platform/bouffalolab/BL616/NetworkCommissioningDriver.h

```diff
@@ -141,6 +141,9 @@ class BLWiFiDriver final : public WiFiDriver
     ConnectCallback * mpConnectCallback;
     NetworkStatusChangeCallback * mpStatusChangeCallback = nullptr;
     int32_t mLastDisconnectedReason;
+
+    char mScanSSID[DeviceLayer::Internal::kMaxWiFiSSIDLength + 1];
```

Comment:
why is there a +1 required? this is not 0-terminated since we have a length.

Reply:
+1 Removed